README.md

```diff
@@ -19,6 +19,14 @@ $ cd git-machete
 $ sudo make install
 ```
 
+Installation could be done by following commands:
+
+```bash
+$ git clone https://github.com/VirtusLab/git-machete.git
+$ cd git-machete
+$ python setup.py install
+```
+
```

Comment:
While you're here, pls also add mention about reloading (sourcing) `~/.bashrc` after the install to enable the Bash completions

Reply:
I forgot to include it earlier

Reply:
I forgot to install `git-machete-prompt`

Reply:
I will update installation of bash completion to have it in user space (no `sudo`).

Reply:
Ok pls lmk when ready I'm curious how it can be done otherwise

Reply:
What I found is to have local `~/.bash_completion` file updated. That file is not default thing in `bash` but it is checked (as per default configuration). IMO completion is also very depending thing. Someone can use `zsh`, `fish` or some other shell.

Anyway I will try to stick to `bash` in this case.

Reply:
There's a task #21 for providing zsh completion as well (seems more wide-spread than `fish` out of those two)